doc/ven_impl/ven_template/README.md

```diff
@@ -0,0 +1,40 @@
+# CNTT Vendor Implementation - VI
+
+## Vendor Name
+Vendor XX
```

Comment:
Should the Vendor information include Vendor Contacts

Reply:
added in https://github.com/cntt-n/CNTT/pull/1305/commits/64eff759468018eb504868f6f57964dd1658338e